www/gatsby-node.js

```diff
@@ -14,7 +14,7 @@ require(`dotenv`).config({
   path: `.env.${process.env.NODE_ENV}`,
 })
 
-if (process.env.NODE_ENV === `production` && !process.env.GITHUB_API_TOKEN) {
+if (!process.env.GITHUB_API_TOKEN) {
   throw new Error(
     `A GitHub token is required to build the site. Check the README.`
```

Comment:
Maybe `www/README` to make it more clear?

Reply:
You're in www, I don't know if this is clearer?

Reply:
But sure! Not too opinionated either way 👍